docs/source/gen/flytectl.rst

```diff
@@ -3,54 +3,54 @@
 flytectl
 --------
 
-flyetcl CLI tool
+**flyetcl** CLI tool
```

Comment:
@evalsocket, is this a suitable file to modify the content? Or, do we have a file in the `cmd` directory?